tox.ini

```
@@ -17,6 +16,7 @@ envlist =
 [testenv]
 skip_install = true
 deps =
+    setuptools < 69
```

I think 69 is still fine, and problems start with 70, so I would suggest this:

```suggestion
    setuptools < 70
```

---

69 is not fine because there's two issues. One is about producing wheels where the wheel name uses underscore characters to replace dots, and for backwards compatibility with buildout we don't want that.